### PR TITLE
PENDING STABILITY: Upgrade libraries

### PR DESCRIPTION
**Pull request by cpennington**
@clintonb, @rlucioni: Review?

### CODE REVIEW

**Comment by clintonb** [comment]
:+1:

**Comment by rlucioni** [comment]
@cpennington what about gevent? The version currently installed is not Python 3 compatible.

**Comment by cpennington** [comment]
Ah, good call. I meant to upgrade that.

**Comment by cpennington** [comment]
I think I'll leave this PR open until I've got the course discovery IDA working in python 3

**Comment by cpennington** [comment]
@rlucioni @clintonb: Re-review?

File: {{cookiecutter.repo_name}}/requirements/production.txt [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Packages required in a production environment
 -r base.txt
 
-gevent==1.0.2
-gunicorn==19.3.0
-MySQL-python==1.2.5
+gevent==1.1rc1
```
Comment (rlucioni) [inline {{cookiecutter.repo_name}}/requirements/production.txt:4]: Is this stable enough that we're comfortable using it for new IDAs?

Comment (cpennington) [inline {{cookiecutter.repo_name}}/requirements/production.txt:4]: Honestly, I'm happy to leave this PR open until I've taken the hit to make sure that course-discovery is stable using the rc (or the released version, if we get there). We also need an unreleased version of gunicorn to make python3 work.

Comment (rlucioni) [inline {{cookiecutter.repo_name}}/requirements/production.txt:4]: I'd prefer going that route to merging this and expecting new IDAs to use an unreleased version of two critical packages.

**Comment by cpennington** [comment]
I'm closing this until we are more sure of the stability of using these libraries for python 3.

**Pull request closed without merging**